pipeline/frontend/yaml/compiler/convert.go

```diff
@@ -16,6 +17,8 @@ import (
 
 func (c *Compiler) createProcess(name string, container *yaml.Container, section string) *backend.Step {
 	var (
+		uuid = uuid.New()
```

Comment:
because the pipeline create the steps ... and the other way is messed up ...

we dont want the pipeline package depend on server, but the pipeline package should assign IDs so that's how it works. we can drop the other id if you like

Reply:
It's in that way a solution and a beginning of a refactor

Reply:
https://github.com/woodpecker-ci/woodpecker/issues/983